packages/protocol/contracts/tokenvault/LibBridgedToken.sol

```diff
@@ -32,4 +51,18 @@ library LibBridgedToken {
     function buildSymbol(string memory symbol) internal pure returns (string memory) {
         return string.concat(symbol, ".t");
     }
+
+    function buildURI(address srcToken, uint256 srcChainId) internal pure returns (string memory) {
+        // Creates a base URI in the format specified by EIP-681:
+        // https://eips.ethereum.org/EIPS/eip-681
+        return string(
+            abi.encodePacked(
+                "ethereum:",
+                Strings.toHexString(uint160(srcToken), 20),
+                "@",
+                Strings.toString(srcChainId),
+                "/tokenURI?uint256="
+            )
+        );
+    }
```

Comment:
Taken from Optimism so that the URI for bridged tokens still points to where the data can be found. Seems reasonable?

Reply:
I just copied it directly from optimism, I haven't read the spec in any detail: https://github.com/ethereum-optimism/optimism/blob/6120d22fc8382f92c990d2c81231fa8e981ed7dd/packages/contracts-bedrock/src/universal/OptimismMintableERC721.sol#L68